docs/deployment/README.md

```diff
@@ -353,18 +353,30 @@ To install the dashboard clone the Cumulus-dashboard repository into the root de
     $ nvm use
     $ npm install
 
-### Dashboard configuration
+#### Dashboard versioning
+
+By default, the `master` branch will be used for dashboard deployments. The `master` branch of the dashboard repo contains the most recent stable release of the dashboard.
+
+If you want to test unreleased changes to the dashboard, use the `develop` branch.
+
+Each release/version of the dashboard will have a tag in the dashboard repo. Release/version numbers will use semantic versioning (major/minor/patch).
```

Comment:
We should probably link to the releases page for the dashboard: https://github.com/nasa/cumulus-dashboard/releases

Which got me thinking... as part of the dashboard release process we should create the release on the releases page.

Reply:
Makes sense. I assume that I need to add that to the documentation for the dashboard then?

Reply:
Here is the corresponding PR to the dashboard repo: https://github.com/nasa/cumulus-dashboard/pull/536